test/spec/duplexBenchmark.js

```diff
@@ -100,6 +100,34 @@ suite.add('compare operation', {
     }
 });
 
+suite.add('compare operation same but deep objects', {
+    setup: function(){
+        var depth = 1;
+        function shallowObj(){
+            return {
+                shallow: {
+                    firstName: "Tomek",
+                    lastName: "Wytrebowicz",
+                    mobileNumbers: [{
+                        number: "12345"
+                    }, {
+                        number: "45353"
+                    }]
+                }
+            };
+        }
+        var obj = shallowObj();
+        var node = obj;
+        while(depth-- > 0){
+            node.nested = shallowObj();
+            node = node.nested;
+        }
+        var obj2 = obj;
```

Comment:
I don't understand this loop. What's the point of it if depth is always 1?
And I don't see/understand the how `node` var is used, too. Maybe if depth was > 1 it would make sense.